### PR TITLE
Make message clearer when specifying default rmw implementation

### PR DESCRIPTION
**Pull request by dhood**
it was a bit vague before about what it was doing

### CODE REVIEW

File: rmw_implementation/rmw_implementation-extras.cmake.in [hunk 1]
```diff
@@ -44,7 +44,7 @@ endif()
 
 
 set(RMW_IMPLEMENTATION "${default_rmw_implementation}")
-message(STATUS "Using RMW implementation: ${RMW_IMPLEMENTATION}")
+message(STATUS "Using RMW implementation '${RMW_IMPLEMENTATION}' as default.")
```
Comment (dirk-thomas) [inline rmw_implementation/rmw_implementation-extras.cmake.in]: Commonly these messages don't have a trailing dot

**Comment by dirk-thomas** [comment]
Beside the minor comment this lgtm.

**Comment by wjwwood** [comment]
Merged with 2ffb86859edd5138d2c948b0912fe0fc18f5979b

**Comment by dhood** [comment]
Thanks for fixing that @wjwwood

**Pull request closed without merging**